Task 7/script.js

```diff
@@ -0,0 +1,18 @@
+/* ------------------------------ TASK 7 -----------------------------------
+Turimas "audi" objektas.
+
+Parašykite funkciją "showObjectKeys", kuri kaip argumentą priims objektą 
+ir grąžins visus jo "key" masyve.
+-------------------------------------------------------------------------- */
+const audi = {
+  make: 'audi',
+  model: 'A6',
+  year: 2005,
+  color: 'white',
+};
+
+const showObjectKeys = (audi) => {
+  return Object.entries(audi).map(item => item[0])
```

Comment:
This works, but i suggest little bit cleaner solution - `return Object.keys(audi);`